js/notifier.js

```diff
@@ -49,13 +49,18 @@ var Notifier = {
     var updated = false;
     var newestProject = sortedProjects[0];
     var newestTimestamp = Date.parse(newestProject.Timestamp);
-    var lastNewestProjectTimestamp = Number(localStorage.getItem("newestProjectTimestamp"));
-    if (lastNewestProjectTimestamp && lastNewestProjectTimestamp < newestTimestamp) {
```

Comment:
fixed a bug here - we forgot to check for the initial load and therefore `newestProjectTimestamp` never got saved to `localStorage` (i.e., this `if` condition we had before could never be met)